Sources/MapboxMaps/Foundation/MapboxObservable.swift

```diff
@@ -71,14 +75,24 @@ internal final class MapboxObservable: MapboxObservableProtocol {
             }
         }
     }
-
+    @available(*, deprecated)
```

Comment:
If we already applied deprecation on Protocol, should we duplicate it in implementation? 🤔 
And if we do, should we also copy-paste `renamed` part?

Reply:
Good point, I missed that, added them 👍